irfragment/src/main/java/com/gigigo/irfragment/reco/ImageRecognitionFragment.kt

```diff
@@ -526,7 +526,7 @@ class ImageRecognitionFragment : Fragment(), IRApplicationControl {
         val result = queryResultsList.at(0)
 
         // Check if this target is suitable for tracking:
-        listener?.onFragmentInteraction(result.targetName)
+        listener?.onScanResult(result.targetName)
```

Comment:
```suggestion
         listener?.onScanResult(result)
```
En la weekly creo que Alberto fue quien dijo que era mejor enviar el result entero